CometServer/Server/GameServer.cpp

```diff
@@ -11,9 +12,9 @@ namespace server
 	//in the class rather than the order in which the members appear in the initializer list.
 	//To avoid confusion, it is best to specify the initializers in the member declaration order."
 	GameServer::GameServer() :
-		universe{R"(universe_initial_state.ssv)"},
-		export_strategy{ client_input_payload_buffer },
-		import_strategy{ server_state_payload_buffer },
+		universe{ "game_data.sqlite3" },
+		export_strategy{ *this },
+		import_strategy{ *this },
```

Comment:
This is convenient because this way we don't need to pass every packet container one-by-one the the export- and import strategies, they will be all accessible through the GameServer instance.